### PR TITLE
add works and about myself section

### PR DESCRIPTION
**Pull request by NoelLincoln**
**Hello reviewer 👋**
**Features added :**

works section and about myself section
used flexbox to layout html elements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,5 +61,276 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+                        product ,
+                        feature or website. Look through some of my work and
+                        experience! If
+                        you like what you see and have a project you need coded,
+                        don’t
+                        hesitate to contact me.</p>
+
+                    <div class="socials">
+                        <div class="support-text">
+                            <p>Let's connect</p>
```
Comment (Olamarx) [inline index.html]: - [x] This is also a header that should use hx, you can consider using h3 or any downward.

File: index.html [hunk 2]
```diff
@@ -61,5 +61,276 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+                  image" /><a href="#">Javascript</a></li>
+                            <li><img src="assets/images/html.png" alt="html
+                  image" /><a href="#">HTML</a></li>
+                            <li><img src="assets/images/javascript.png"
+                                    alt="javascript
+                  image" /><a href="#">CSS</a></li>
+                        </ul>
+                    </ul>
+                    <ul class="dropdown">
+                        <input type="checkbox" id="dropdown" />
```
Comment (Olamarx) [inline index.html:317]: - [x] Kindly note that one of the advantages of ul li is for code organization, once you have used ul, the child should be li so this ul should be in li tags each too.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @NoelLincoln, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️ 
- Grid and Flexbox approach ✔️
- Commit messages meaningful ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (NoelLincoln) [reply index.html:317]: the li tags are the children of the unordered list tag, then the list items wrap the anchor tags. Kindly expound

**Review by skyv26 — APPROVED** [review]
## Hi @NoelLincoln, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Linters are green ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**